autoheal-dev.yml

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is an example configuration file used for development.
+
+#
+# This section contains the details needed to connect to the AWX or Ansible
+# Tower server that will execute the Ansible playbooks.
+#
+awx:
```

Comment:
Even for development environments you will need at least the `crendentials` section:

```yaml
credentials:
  username: myuser
  password: mypassword
```

Otherwise the connection the AWX server will not work.

Reply:
:+1: done